training-service/src/app/pages/about/about.component.css

```diff
@@ -1,4 +1,19 @@
-.col-sm{
-    display: table;
-    margin:auto;
-}
+.image-padding {
+    width: 100%;
+    padding-top: 1rem;
+  }
+  
+  .content-padding {
```

Comment:
classes names should point to the element not to a property, 
instead of content-padding 
should be content etc

Reply:
done